### ISSUE TITLE
NoAccessors.NoSetter - wrong warning for "non-setters"

### ISSUE THREAD

**Issue by Chrico** [inpsyde/php-coding-standards#6]
I've following error message with the latest 0.7.x - release:

```
Setters are discouraged. Try to use immutable objects, constructor injection and for objects that really needs changing state try behavior naming instead, e.g. changeName() instead of setName(). InpsydeCodingStandard.CodeQuality.NoAccessors.NoSetter)
```

code is following:

```php
/**
 * @return array
 */
public function settingsSpec(): array
{
	// snip
}
```

It seems the quality tool does match by the keyword `set`, which is contained in `settingsSpec`.

**Comment by gmazzap** [comment]
Bug fixed already in master aa68a3da88094a8e6f45e03bd406ff38c4450841

**Comment by gmazzap** [comment]
Releasee v0.8.0 which includes this fix.

**Comment by gmazzap** [comment]
Closing this now that is fixed and new version released.
